ready_recipes/arm_pyart/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [ `uname` == Darwin ]; then
+    # Remove gfortran so that all fortran Py-ART modules are not built.  This
+    # avoids the need to package the gfortran run time in the conda package.
```

Comment:
There is a libgfortran provided by defaults, no? https://anaconda.org/anaconda/libgfortran
Looks like it is only Linux though...